Cargo.toml

```
@@ -4,11 +4,11 @@ version = "0.1.0"
 edition = "2021"
 rust-version = "1.73"
 authors = ["Joshua Thijssen <jaytaph@deadcode.nl>"]
-description = "html5 browser engine"
+description = "A modern HTML5 browser engine written in Rust."
```

I kind of want to avoid selling this project as being bigger than it is at this point.  Even if we strive for a "modern" brower, I think we're years away from that.  So I'd pitch for going with understatement here.

```suggestion
description = "An HTML5 browser engine written in Rust."
```